packages/datadog-plugin-langchain/src/handlers/chain.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const LangChainHandler = require('./default')
+
+class LangChainChainHandler extends LangChainHandler {
+  getSpanStartTags (ctx) {
+    const tags = {}
+
+    if (!this.isPromptCompletionSampled()) return tags
+
+    let inputs = ctx.args?.[0]
+    inputs = Array.isArray(inputs) ? inputs : [inputs]
+
+    for (const idx in inputs) {
+      const input = inputs[idx]
+      if (typeof input !== 'object') {
+        tags[`langchain.request.inputs.${idx}`] = this.truncate(input)
+      } else {
+        for (const [key, value] of Object.entries(input)) {
+        // these are mappings to the python client names, ie lc_kwargs
+        // only present on BaseMessage types
+          if (key.includes('lc')) continue
```

Comment:
they are internal fields, and often don't hold relevant information (there's a field called `lc_serializable`, which is just a boolean not relevant to the chain), or have duplicate information (ie `lc_namespace`, which we use for the resource name, or `lc_kwargs`, which are duplicates of some of the input values, which we already tag).

while some of them actually won't pass the `truncate`/normalize check, since they are not strings, i added this to be safe/not add useless tags. can add them on down the road if there's a use case for them!

Reply:
ah ok I see! should the string check be `startsWith('lc_')` then?

Reply:
i think that's fair!